DeepSpeech/Dockerfile.train

```diff
@@ -17,6 +17,7 @@ ARG lm_alpha=0.75
 ARG lm_beta=1.85
 ARG early_stop=1
 
+ARG lm_evaluate_range=
```

Comment:
with no value?

Reply:
Good point! Actually the french repo is in this way. I'll look at it properly

Reply:
@lysyx which value should we use here?

Reply:
from the french repo you can see you can give the param at runtime using `docker run -e lm_evaluate_range=[list of values]`. I don't if it makes sense to have a different default

Reply:
let's try with them, we can always change them later

Reply:
Well, yeah, no value, because LM evaluation takes much more time, so I thought it would make sense that the user does:
 - first train with lm_alpha/lm_beta set to 0.0
 - second, perform a run keeping the checkpoints but setting lm_evaluate_range

Then after analysis of the run, one can decide what values to set for `lm_alpha` and `lm_beta`